### PR TITLE
8.4.1

### PR DESCRIPTION
**Pull request by arnaumartifont**

### CODE REVIEW

File: Swift/ChartboostExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -295,22 +334,22 @@
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.chartboost.ChartboostExampleSwift;
-				PRODUCT_NAME = Chartboost;
+				PRODUCT_NAME = "$(TARGET_NAME)";
```
Comment (daniel-barros) [inline Swift/ChartboostExample.xcodeproj/project.pbxproj:337]: Why this change?

File: Swift/ChartboostExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7,30 +7,30 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		342BBF295473A28B87B0C20A /* Pods_ChartboostExample_Pods.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 62F672CEC8AE474CE80A0CCF /* Pods_ChartboostExample_Pods.framework */; };
```
Comment (daniel-barros) [inline Swift/ChartboostExample.xcodeproj/project.pbxproj:10]: I see a lot of changes in the Swift xcodeproj but none on the Obj-C xcodeproj. Is there a reason for this?

**Review by daniel-barros — CHANGES_REQUESTED** [review]
The readme is outdated

File: Swift/ChartboostExample.xcodeproj/project.pbxproj (hunk 2 above)
Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:10]: yes, there was a missing target in the Swift project

File: Swift/ChartboostExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:337]: yea I know, now the name is changed but it shows via the plist, and this isn't updated but is working, weird

**Comment by arnaumartifont** [comment]
> The readme is outdated

i'm on it 🙌

File: Swift/ChartboostExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:337]: this was related adding the missing target back, it was already created the 'ChartboostExample' but somehow got unchecked to be used, so adding it back it changed this since having the two targets now

Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:337]: i'm gonna revert it, and remove the last commit, should work

**Review by daniel-barros — APPROVED** [review]
